Sources/MapboxCoreNavigation/PassiveLocationDataSource.swift

```diff
@@ -85,6 +98,21 @@ open class PassiveLocationDataSource: NSObject {
             }
         }
     }
+
+    /**
+     Sets electronic horizon options. Pass `nil` to reset to defaults.
+     */
+    public func set(electronicHorizonOptions: ElectronicHorizonOptions?) {
+        try! navigator.setElectronicHorizonOptionsFor(electronicHorizonOptions)
+    }
+
+    public var graphAccessor: GraphAccessor
+
+    public var roadObjectsStore: RoadObjectsStore {
```

Comment:
These classes need Swift wrapper objects, or perhaps their important methods could be exposed by the Navigator class that would be created in #2823 (not to be confused with `MBNNNavigator`).

Reply:
Right, wrappers are introduced in the latest commits.